OpenEdXMobile/src/main/java/org/edx/mobile/model/discussion/DiscussionTextUtils.java

```diff
@@ -1,4 +1,4 @@
-package org.edx.mobile.discussion;
+package org.edx.mobile.model.discussion;
```

Comment:
It's a Util class and should not be in Model PAckage.

Reply:
Oops, Sorry, Change reverted